CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 Changelog for package pcl_msgs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+ROS2 (2018-12-31)
```

Comment:
changelogs are generated at release time, please remove this

Reply:
Ok, Remove the same.